JellyfinPlayer tvOS/Views/LiveTVChannelsView.swift

```diff
@@ -0,0 +1,104 @@
+//
+ /*
+  * SwiftFin is subject to the terms of the Mozilla Public
+  * License, v2.0. If a copy of the MPL was not distributed with this
+  * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+  *
+  * Copyright 2021 Aiden Vigue & Jellyfin Contributors
+  */
+
+import Foundation
+import JellyfinAPI
+import SwiftUI
+import SwiftUICollection
+
+
+struct LiveTVChannelsView: View {
+    @EnvironmentObject var router: LiveTVChannelsCoordinator.Router
+    @StateObject var viewModel = LiveTVChannelsViewModel()
+    
+    var body: some View {
+        if viewModel.isLoading == true {
+            ProgressView()
+        } else if !viewModel.rows.isEmpty {
+            CollectionView(rows: viewModel.rows) { section, env in
+                return createGridLayout()
+            } cell: { indexPath, cell in
+                makeCellView(indexPath: indexPath, cell: cell)
+            } supplementaryView: { _, indexPath in
+                EmptyView()
+                .accessibilityIdentifier("\(indexPath.section).\(indexPath.row)")
+            }
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+            .ignoresSafeArea()
+            
+        } else {
+            VStack {
+                Text("No results.")
+                Button {
+                    print("movieLibraries reload")
+                } label: {
+                    Text("Reload")
+                }
+            }
+        }
+    }
+    
+    @ViewBuilder func makeCellView(indexPath: IndexPath, cell: LiveTVChannelRowCell) -> some View {
+        GeometryReader { _ in
```

Comment:
Is there a reason to use a reader here without using the value of it?

Reply:
No- this wasn't actually needed. Removed in latest commit.